mne/tests/test_annotations.py

```diff
@@ -86,8 +87,22 @@ def test_annotations():
     assert_array_equal(raw.annotations.description, ['x', 'x', 'x', 'x', 'x',
                                                      'x', 'y'])
 
-    # Test concatenating annotations with and without orig_time.
+    # Test cropping with annotations
     raw = read_raw_fif(fif_fname)
+    events = mne.find_events(raw)
+    onset = events[events[:, 2] == 1, 0] / raw.info['sfreq']
+    duration = np.ones_like(onset)  # 1s
+    description = ['bad %d' % k for k in range(len(onset))]
+    raw.annotations = mne.Annotations(onset, duration, description,
+                                      orig_time=raw.info['meas_date'])
+
+    raw_cropped = raw.copy().crop(raw.times[-1] / 2. + 2., None)
+    assert_array_equal(raw_cropped.annotations.description,
+                       raw.annotations.description[len(onset) // 2:])
```

Comment:
shouldn't we also check that the `onset` is still the same (`assert_allclose` at least)?

How about checking that if you add annotations, crop, and concatenate, that the annotations are equal?

Reply:
FWIW the test suggested here fails on this PR.